src/org/zalando/stups/magnificent/api.clj

```diff
@@ -0,0 +1,154 @@
+(ns org.zalando.stups.magnificent.api
+  (:require [org.zalando.stups.friboo.system.http :refer [def-http-component]]
+            [org.zalando.stups.magnificent.util :as util]
+            [ring.util.response :as ring]
+            [org.zalando.stups.friboo.log :as log]
+            [clojure.core.async :refer [put! chan <!! go]]
+            [io.sarnowski.swagger1st.util.api :as api]
+            [org.zalando.stups.magnificent.external.team :as team]
+            [org.zalando.stups.magnificent.external.user :as user]
+            [org.zalando.stups.magnificent.external.account :as account]
+            [org.zalando.stups.friboo.ring :as fring]
+            [io.sarnowski.swagger1st.util.api :as api]))
+
+(def-http-component API "api/magnificent-api.yaml" [])
+(def default-http-configuration {:http-port 8080})
+
+(defn- find-user
+  [realm user request]
+  (let [user-api (get-in request [:configuration :user-api])
```

Comment:
is user-api / kio-api a url? If yes, please consider suffixing the variable names with "-url"